corehq/apps/geospatial/static/geospatial/js/gps_capture.js

```diff
@@ -219,6 +221,42 @@ hqDefine("geospatial/js/gps_capture",[
             self.isCreatingCase(true);
             const caseToCreate = new dataItemModel(null, self.dataType);
             self.captureLocationForItem(caseToCreate);
+
+            $("#owner-select").select2({
+                placeholder: gettext('Current User'),
+                cache: true,
+                allowClear: true,
+                delay: 250,
+                ajax: {
+                    url: initialPageData.reverse('paginate_mobile_workers'),
+                    dataType: 'json',
+                    data: function (params) {
+                        return {
+                            query: params.term,
+                            page_limit: USER_LIMIT,
+                            page: params.page,
+                        };
+                    },
+                    processResults: function (data, params) {
+                        params.page = params.page || 1;
+
+                        const hasMore = (params.page * USER_LIMIT) < data.total;
+                        const dataResults = $.map(data.users, function (user) {
+                            return {
+                                text: user.username,
+                                id: user.user_id,
+                            };
```

Comment:
nit: I guess if you simply use username and id as the properties, which are reasonable property names, you won't need to do the mapping.

Reply:
I originally tried to do that, as that would be ideal. Unfortunately however, it seems the select2 widget is insistent that it be in this format. Changing it results in no entries being shown on the dropdown. Looking at some other JS examples in the codebase, it is also being done in this format.

Reply:
aha, thanks for following up on the feedback